django_restql/mixins.py

```diff
@@ -18,36 +18,26 @@
 )
 
 
-class DynamicFieldsMixin(object):
-    query_param_name = "query"
-
-    def __init__(self, *args, **kwargs):
-        # Don't pass 'query', 'fields' and 'exclude' kwargs to the superclass
-        self.query = kwargs.pop('query', None)  # Parsed query
-        self.allowed_fields = kwargs.pop('fields', None)
-        self.excluded_fields = kwargs.pop('exclude', None)
-        self.return_pk = kwargs.pop('return_pk', False)
-
-        is_field_kwarg_set = self.allowed_fields is not None
-        is_exclude_kwarg_set = self.excluded_fields is not None
-        msg = "May not set both `fields` and `exclude`"
-        assert not(is_field_kwarg_set and is_exclude_kwarg_set), msg
-
-        # Instantiate the superclass normally
-        super().__init__(*args, **kwargs)
-
-    def to_representation(self, instance):
-        if self.return_pk:
-            return instance.pk
-        return super().to_representation(instance)
+class RequestQueryParserMixin(object):
```

Comment:
Since this is going to be the base for the `EagerLoadingMixin` on DRF views, a lot of these static/class method names should probably be pretty removed from what someone might confuse as a built-in to DRF.

Here are some suggestions:

`get_query_param_name` -> `get_restql_query_param_name`
`has_query_param` -> `has_restql_query_param`
`get_raw_query` -> `get_raw_restql_query`
`get_parsed_query_from_req` -> `get_parsed_restql_query_from_req`